test/test_p2p.py

```diff
@@ -84,6 +111,25 @@ async def test():
                 await self.client.call_method_async(None, '/test', 'cockpit.Test', 'DivideSlowly', 'ii', 1554, 0)
         run_async(test())
 
+    def test_signal(self):
+        async def test():
+            # XXX - without this initial pointless method call, the
```

Comment:
I am debugging this.  The server side gets stuck during authentication somehow (without this call) and then it all times out after a while.  I am confident that I can figure this out, let's see...

Reply:
https://github.com/systemd/systemd/pull/24875